layouts/_default/content.html

```diff
@@ -0,0 +1,36 @@
+{{/* Copied and adapted from themes/docsy/layouts/_default/content.html */}}
+<div class="td-content">
+    {{- if site.Params.automaticSectionNumbers -}}
```

Comment:
This is the only part that I changed. The rest is copied from https://github.com/acend/docsy-plus